scripts/setup_db.py

```diff
@@ -33,45 +37,39 @@ def create_tables(hostname, dbname, user):
             )
             """,
             """
-            DROP TABLE IF EXISTS service_providers
-            """,
-            """
             CREATE TABLE service_providers (
-                id INTEGER PRIMARY KEY,
-                census_division_id INTEGER REFERENCES census_division (id),
-                name TEXT NOT NULL,
-                website TEXT,
-                report_year INTEGER,
-                report_link TEXT,
-                expenses INTEGER,
+                id           INTEGER PRIMARY KEY,
+                name         TEXT NOT NULL,
+                website      TEXT,
```

Comment:
should this be `NOT NULL`? is there value in having service providers that C4K can't look into